### PR TITLE
added pending_payment state in task-warrant

### PR DESCRIPTION
**Pull request by nileshgupta111**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new workflow state, "PENDING_PAYMENT," requiring document uploads and featuring a Service Level Agreement (SLA).
- **Updates**
	- Restored the "business" field to the "BusinessServices" array.
	- Modified action transitions for the "CREATE" action to include the new "PENDING_PAYMENT" state.
	- Adjusted properties of terminal states to ensure consistency in state management.
	- Updated the "PENDING_PAYMENT" state to no longer be a starting point in the workflow for both "task-notice" and "task-summons" services.

### CODE REVIEW

File: docs/task/workflow/workflowConfig-warrant.json [hunk 1]
```diff
@@ -17,13 +17,42 @@
...
               ]
             }
           ]
         },
+        {
+          "sla": 172800000,
+          "state": "PENDING_PAYMENT",
+          "applicationStatus": "INWORKFLOW",
+          "docUploadRequired": true,
+          "isStartState": true,
```
[the remark this answers is not in the document]
Comment (atulgupta2024) [reply docs/task/workflow/workflowConfig-warrant.json:32]: @nileshgupta111 isStartSate for PENDING_PAYMENT  should be false.

**Review by atulgupta2024 — CHANGES_REQUESTED** [review]
1 minor fix required

File: docs/task/workflow/workflowConfig-warrant.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nileshgupta111) [reply docs/task/workflow/workflowConfig-warrant.json]: @atulgupta2024  updated isStartSate for PENDING_PAYMENT in warrant notice and summons

**Review by atulgupta2024 — APPROVED** [review]
reviewed changes

**Pull request merged**